src/Appwrite/Platform/Workers/Deletes.php

```diff
@@ -553,6 +556,8 @@ private function deleteProject(Database $dbForConsole, callable $getProjectDB, D
         // Delete metadata table
         if ($dsn->getHost() !== DATABASE_SHARED_TABLES) {
             $dbForProject->deleteCollection('_metadata');
+        } else {
+            $this->deleteByGroup('_metadata', [], $dbForProject);
```

Comment:
Doesn't this delete all documents form the _metadata table in the project db? 🧐

Reply:
It only deletes the documents that the current tenant can access, in shared tables mode, all queries have `WHERE _tenant = :tenant` appended